src/main.rs

```diff
@@ -33,29 +37,33 @@ impl Terminal {
         }
     }
 
-    fn ask_for_new_todo(&mut self) -> Todo {
+    fn ask_for_new_todo(&mut self) -> Result<Todo, Error> {
         writeln!(self.stdout, "\nQuer adicionar um novo TODO 📝?\ndigite (sim) para confirmar 👍  ou (nao) para negar 👎").unwrap();
-        
+
         let mut resposta = input();
```

Comment:
Seguindo a linha de raciocínio de retirar a  função `input()`, aqui você pode usar `String::new()`.
```suggestion
        let mut resposta = String::new();
```

Reply:
feito!